File: MODUL 3/Soal 4/PRAK304_2410817210026-Fadhil_Syahdama_Mahatma_Putra.c
#include <stdio.h>
int main(){
    int bocchi;

    scanf("%d", &bocchi);

    if(bocchi==0){
        printf("Nol");
    } else if(bocchi>0 && bocchi<10){
        printf("Satuan");
    } else if(bocchi>10 && bocchi<20){
        printf("Belasan");
    } else if(bocchi>=20 && bocchi<100){
        printf("Puluhan");
    }
    else if(bocchi>=100){
        printf("Anda Menginput Melebihi Limit Bilangan ");
    }
}

File: MODUL 3/Soal 4/PRAK304_2410817210026-Fadhil_Syahdama_Mahatma_Putra.py
def main():
    albedo = int(input())

    if(albedo==0):
        print("Nol")
    elif(albedo>0 and albedo<10):
        print("Satuan")
    elif(albedo>10 and albedo<20):
        print("Belasan")
    elif(albedo>=20 and albedo<100):
        print("Puluhan")
    elif(albedo>=100):
        print("Anda Menginput Melebihi Limit Bilangan ")
        
main()